src/goto-checker/symex_coverage.cpp

```diff
@@ -438,3 +438,51 @@ bool symex_coveraget::generate_report(
     return output_report(goto_functions, out);
   }
 }
+
+
+const goto_programt::const_targetst symex_coveraget::coverage_to (goto_programt::const_targett from) const
+{
+  std::list<goto_programt::const_targett> to_list;
+  const auto &inner = coverage.find(from);
+  if (inner != coverage.end()) 
+  {
+    const std::map<goto_programt::const_targett, coverage_infot> &to_map = inner->second;
+    for (auto it = to_map.begin(); it != to_map.end(); it++)
```

Comment:
Is not ++it preferred to it++ in C++ (when it doesn't matter, as it doesn't here?) as it avoids a temporary copy?

Reply:
You'd think the compiler would be clever enough to not do that. Anyways, updated across the code changes.